src/icons/Torso.tsx

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable react/display-name */
```

Comment:
if this is replacing the one in ui/icons, please remove that one

Reply:
thanks for not adding "Icon" to the end of the filename -- i find that sort of thing redundant. but we should be consistent, so with SettingsIcon also in this PR, let's have it just be one way or the other

Reply:
All of our stitches icons have the `/* eslint-disable react/display-name */` :( What is the proper way to add display name?   I'm not seeing any examples yet

Reply:
```js
Torso.displayName = 'Torso';
```